packages/relay/tests/lib/eth/eth_call.spec.ts

```diff
@@ -135,6 +135,7 @@ describe('@ethCall Eth Call spec', async function () {
     });
 
     it('eth_call with incorrect `to` field length', async function () {
+      web3Mock.onPost('contracts/call').reply(200);
```

Comment:
QQ: Why are we mocking this here?

Reply:
I made a mistake. Some of the tests show this error
 
![Screenshot 2024-04-19 at 12 39 53](https://github.com/hashgraph/hedera-json-rpc-relay/assets/42917570/53eef587-2f75-4aa6-82ff-8b27171434ec)
However, I fixed it in the wrong test, in reality the one below this one is missing this mock. Will take a look and fix it